src/m365/spo/commands/eventreceiver/eventreceiver-remove.ts

```diff
@@ -52,10 +53,10 @@ class SpoEventreceiverRemoveCommand extends SpoCommand {
         listId: typeof args.options.listId !== 'undefined',
         listTitle: typeof args.options.listTitle !== 'undefined',
         listUrl: typeof args.options.listUrl !== 'undefined',
-        scope: typeof args.options.scope !== 'undefined',
+        scope: args.options.scope,
```

Comment:
```suggestion
        scope: args.options.scope || 'site',
```
@milanholemans when not defined I will log the `site` scope as this seems to be the default one when this is left undefined.

Reply:
If I check the docs, `web` should be the default value.

> The scope. Can be either "site" or "web". Defaults to "web". Only applicable when not specifying any of the list properties.

Thank you for fixing this again ❤️
I totally forgot to fix it 😁

Reply:
sorry... web... totally got that the other way around looking at the old conditions before your change 😜.
no problem 👍